### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Satyr09**
Changed the way sections are linked.
Fixed markdown in resources section

### CODE REVIEW

**Comment by athityakumar** [comment]
@Satyr09 - Thanks for working on this. I can still see a few HTML tags in the README here and there. Can you completely remove all the HTML tags and make it into a pure markdown README?

**Comment by Satyr09** [comment]
Sure.
Will do.

File: README.md [hunk 1]
```diff
@@ -396,25 +388,27 @@ git.rb
...
-[Back to contents](#top)
-<br><br>
+
+## License
+
+[Back to contents](#list-of-contents)
+ 
 The MIT License (MIT)
 
 Copyright (c) 2016 Athitya Kumar
```
Comment (athityakumar) [inline README.md]: Change this to `The MIT License Copyright (c) 2017 - [Athitya Kumar](https://github.com/athityakumar). Please have a look at the [LICENSE.md](LICENSE.md) for more details.`

File: README.md [hunk 2]
```diff
@@ -65,11 +61,11 @@ Website is accessible at https://athityakumar.github.io/tvseries/index.html
...
-<h1> <u> Working of the scripts </u> </h1>
 
-[Back to contents](#top)
-<br><br>
+## Working of the scripts
+
+[Back to contents](#list-of-contents)
+
 The `auto/ruby/index.rb` and the `auto/data/index.json` files are the crucial players here. This is the step-by-step working of this repository :
 <br> (1) `auto/ruby/index.rb` reads the `auto/data/index.json` file and knows which website(s) to look up for which series.
```
Comment (athityakumar) [inline README.md]: Remove the `<br>`s please. 😄

**Comment by Satyr09** [comment]
Hopefully,that's okay now.

File: README.md [hunk 3]
```diff
@@ -60,91 +61,94 @@ Website is accessible at https://athityakumar.github.io/tvseries/index.html
 
 <br>Feel free to add more series' to the list by sending Pull Requests.
```
Comment (athityakumar) [inline README.md]: Remove all the `<br>` tags, please.

Comment (Satyr09) [reply README.md]: Sorry for the carelessness .
Please tell me if its fine now.

**Review by athityakumar — APPROVED** [review]
Good job! Thanks again for contributing. I'm merging this in. 👍

**Comment by athityakumar** [comment]
@Satyr09 - If you're interested, you can also have a look at #44 and change the README template accordingly. 😄

**Comment by Satyr09** [comment]
Will look into it.
Thanks

**Pull request merged**